app/api-requests/rpa-api/does-not-have-any-valid-cph.js

```diff
@@ -0,0 +1,8 @@
+class DoesNotHaveAnyValidCph extends Error {
```

Comment:
any reason this has replaced the NoEligibleCphError within the exceptions folder, which I think is a better place to hold it?